src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/create.html.twig

```diff
@@ -0,0 +1,49 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
+{# It is very important to use product theme ONLY or the default theme from Symfony config is always preloaded #}
+{% form_theme productForm with ['@PrestaShop/Admin/Sell/Catalog/Product/FormTheme/product.html.twig'] only %}
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="{{ asset('themes/new-theme/public/product.css') }}" type="text/css" media="all">
+{% endblock %}
+
+{% block content %}
+  {{ form_start(productForm, {'attr': {'class': 'form-horizontal create-product-form product-page-v2 justify-content-md-center', 'novalidate': 'novalidate'}}) }}
```

Comment:
Why not define CSS classes and the other attribute in PHP Form Class ?

Reply:
After checking more closely I'm not sure this should be part of the form type, these classes are only used for CSS, they are purely used for design so it seems more logical to add them in the layout part (twig)